setup.py

```diff
@@ -57,8 +58,19 @@ def finalize_options(self) -> None:
         mkpath(self.build_temp)
         copy_file(join(dirname(__file__), 'CMakeLists.txt'),
                   self.build_temp)
-        cmake = run(['cmake', '.'], check=True, stdout=DEVNULL, stderr=PIPE,
-                    cwd=self.build_temp, universal_newlines=True)
+        try:
+            cmake = run(['cmake', '.'],
+                        check=True, stdout=DEVNULL, stderr=PIPE,
+                        cwd=self.build_temp, universal_newlines=True)
+        except CalledProcessError as e:
+            print(f"\n\ncmake: Error occurred!\n\n"
+                  f"(Standard output)\n"
+                  f"{e.stdout}\n\n"
+                  f"(Standard error)\n"
+                  f"{e.stderr}\n",
+                  file=sys.stderr)
```

Comment:
In this project, we prefer to use single quotation mark `'` to double quotation mark `"`